_inc/lib/debugger/class-jetpack-cxn-tests.php

```diff
@@ -337,10 +337,23 @@ protected function last__wpcom_self_test() {
 
 		remove_filter( 'http_request_timeout', array( 'Jetpack_Debugger', 'jetpack_increase_timeout' ) );
 
+		$error_msg = wp_kses(
+			sprintf(
+				/* translators: 1st placeholder is a link to Jetpack debug page, 2nd one fetches current site URL. */
+				__(
+					'<a href="%1$s%2$s">Visit the Jetpack.com debug page</a> for more information or <a href="https://jetpack.com/contact-support/">contact support</a>.',
+					'jetpack'
+				),
+				'https://jetpack.com/support/debug/?url=',
```

Comment:
If we do just the one place holder, we condense these two into arguments (the debug URL and the site URL) into one:

```php
esc_url( add_query_arg( 'url', urlencode( site_url() ), 'https://jetpack.com/support/debug/' ) )
```

This also ensures we're using the right escaping functions (`esc_url()` for the whole URL, `urlencode()` for the query string parameter) in the right places.

Reply:
Oh! didn't think of that. Also `add_query_arg` is a TIL for me. Thanks. Made the change now.